web3.js/src/connection.ts

```diff
@@ -2998,6 +2998,24 @@ export class Connection {
     };
   }
 
+  /**
+   * Fetch the fee for a message from the cluster, return with context
+   */
+  async getFeeForMessage(
+    message: Message,
+    commitment?: Commitment,
+  ): Promise<RpcResponseAndContext<number | null>> {
+    const wireMessage = message.serialize().toString('base64');
+    const args = this._buildArgs([wireMessage], commitment);
+    const unsafeRes = await this._rpcRequest('getFeeForMessage', args);
+
+    const res = create(unsafeRes, jsonRpcResultAndContext(nullable(number())));
+    if ('error' in res) {
+      throw new Error('failed to get slot: ' + res.error.message);
+    }
+    return res.result;
```

Comment:
Can you throw an error here if the result is null? It means the blockhash was invalid

Reply:
Oh, missed this in rust code. Fixed. Thanks!